skore/src/skore/sklearn/_estimator/metrics_accessor.py

```diff
@@ -63,8 +63,15 @@ def report_metrics(
         data_source: DataSource = "test",
         X: Optional[ArrayLike] = None,
         y: Optional[ArrayLike] = None,
-        scoring: Optional[list[Union[str, Callable, SKLearnScorer]]] = None,
-        scoring_names: Optional[list[Union[str, None]]] = None,
+        scoring: Optional[
+            Union[
+                str,
+                Callable[..., object],
+                SKLearnScorer,
+                list[Union[str, Callable[..., object], SKLearnScorer]],
+            ]
+        ] = None,
```

Comment:
Define a type to make this more readable, e.g.
```suggestion
        scoring: Optional[Union[Scoring, list[Scoring]]] = None,
```